keyboards/varys/rules.mk

```diff
@@ -0,0 +1,5 @@
+ENCODER_ENABLE = yes
+OLED_ENABLE = yes
+OLED_DRIVER = ssd1306
+OLED_TRANSPORT = i2c
```

Comment:
These are the default, and don't need to be set, despite what stuff like chatgpt says. 
```suggestion
```